opentelemetry-sdk/src/testing/logs/in_memory_exporter.rs

```diff
@@ -107,13 +124,20 @@ impl InMemoryLogsExporter {
     /// let emitted_logs = exporter.get_emitted_logs().unwrap();
     /// ```
     ///
-    pub fn get_emitted_logs(&self) -> LogResult<Vec<LogData>> {
-        self.logs
-            .lock()
-            .map(|logs_guard| logs_guard.iter().cloned().collect())
-            .map_err(LogError::from)
-    }
+    pub fn get_emitted_logs(&self) -> LogResult<Vec<LogDataWithResource>> {
```

Comment:
maybe we can keep the existing public api, and add a new api for get_resource.

Reply:
The get_emitted_logs() still returns the data in the same (older LogData) format, only the struct name is now `LogDataWithResource` instead of `LogData`.